src/main/kotlin/com/getkeepsafe/dexcount/Plugin.kt

```diff
@@ -137,6 +140,14 @@ abstract class TaskProvider(
                 ext!!.libraryVariants
             }
 
+            project.plugins.hasPlugin(JavaPlugin::class.java) || project.plugins.hasPlugin(JavaLibraryPlugin::class.java) -> {
```

Comment:
I wouldn't. If you use Kotlin on the JVM, then you must also apply the Java plugin AFAIK. I don't how this would change with Kotlin multiplatform, though. I'm a little hesitant.

Reply:
Yeah, that's what I was thinking.  I hadn't considered multiplatform, and I'm not inclined to accommodate it now either.